internal/types/array.go

```diff
@@ -111,3 +111,38 @@ func (a *Array) Append(values ...any) error {
 func (a *Array) RemoveByPath(path Path) {
 	removeByPath(a, path)
 }
+
+// Contains checks if the Array contains the given value.
+func (a *Array) Contains(value any) bool {
+	for _, v := range a.s {
+		if v == value {
+			return true
+		}
+	}
+	return false
+}
+
+// ContainsAll checks if the Array contains all the given values of the Array b.
+// Currently, this algorithm is O(n^2) without any performance tuning.
+// Possible improvement: to sorts both arrays before comparing.
+// If sorting is implemented, calling of the Contains function will be O(n*log(n)) instead of O(n^2).
+func (a *Array) ContainsAll(b *Array) bool {
+	if a.Len() < b.Len() {
+		// TODO: How to validate that all the values in b are unique?
+		return false
+	}
+
+	// TODO: This place can be significantly improved:
```

Comment:
@AlekSi as I mentioned before, currently this algorithm is super slow. If you think we need to optimize it right now, please choose what optimization we should use.

Reply:
I think it is okay as it is for now. We will be able to see that performance problem during benchmark profiling – once we have it :)